jenkins-scripts/dsl/test.dsl

```diff
@@ -16,3 +16,28 @@ OSRFLinuxCompilationAnyGitHub.create(ignition_ci_pr_job,
                                      false,
                                      false,
                                      ['main'])
+
+// -------------------------------------------------------------------
+def outdated_job_runner = job("_outdated_job_runner")
```

Comment:
Same job name in test.dsl than in core.dsl?

Reply:
I added `_test` prefix to the job in test.dsl